silver/tests/unit/test_transaction.py

```diff
@@ -0,0 +1,45 @@
+from django.test import TestCase
+from django_fsm import TransitionNotAllowed
+from silver.models import Transaction, Payment
+from silver.tests.factories import TransactionFactory, PaymentFactory
+
+TStates = Transaction.State
+PStates = Payment.Status
+
+class TestTransactionStates(TestCase):
+
+    def test_transaction_succeed(self):
+        payment = PaymentFactory.create()
+        transaction = TransactionFactory.create(payment=payment)
+        transaction.pending()
```

Comment:
You are also testing the pending transition in this test. Do a separate test for that and use `transaction.state = transaction.States.Pending` in this one.